app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] let's remove commented code as it makes code hard to read.

Reply:
Following commit for the solution:

- https://github.com/Linushaddai99/Group-Recipe-app/pull/33/commits/fc628162f43d3f2c3eb1ad3554a036b4bfde7a22